apps/statistics/flow-query-certificates-pdf-conf.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var db                = require('../../db')
+  , assign            = require('es5-ext/object/assign')
+  , dateToParamConfig = require('../../apps-common/query-conf/date-to');
+
+module.exports = [
+	require('../../apps-common/query-conf/date-from'),
+	assign({},
+		dateToParamConfig,
+		{
+			ensure: function (value) {
+				if (!value) return new db.Date();
+				return dateToParamConfig.ensure(value);
```

Comment:
Why do we need such customization just for PDF case? Seems inconsistent. Is there any reason query handlers should be different for HTML, PDF and CSV?